apps/emqx_authz/src/emqx_authz_api_sources.erl

```diff
@@ -75,10 +79,7 @@ schema("/authorization/sources") ->
                 tags => ?TAGS,
                 responses =>
                     #{
-                        200 => mk(
-                            array(hoconsc:union(authz_sources_type_refs())),
-                            #{desc => ?DESC(sources)}
-                        )
+                        200 => ref(?MODULE, sources)
```

Comment:
this reads like: response body should be something like `{"source": []}`?
if yes, it contradicts the test case `?assertEqual([], get_sources(Result1)),`

Reply:
nvm, now I realise `get_sources` is implemented like this: `maps:get(<<"sources">>, jsx:decode(Result), []).`